Refine the “Template literals” article (#13995)
mdn/content: files/en-us/web/javascript/reference/template_literals/index.md

---
{{JsSidebar("More")}}

Template literals are literals delimited with backtick (\`) characters, allowing for [multi-line strings](#multi-line_strings), for [string interpolation](#string-interpolation) with embedded expressions, and for special constructs called [tagged templates](#tagged-templates).

> Template literals are sometimes informally called _template strings_, because one of their most common uses is to create strings by substituting placeholder variables and expressions and returning a string ([string interpolation](#string-interpolation)). 

Reviewer: ```suggestion
Template literals are sometimes informally called _template strings_, because one of their most common uses is to create strings by substituting placeholder variables and expressions ([string interpolation](#string-interpolation)). 
```

---
{{JsSidebar("More")}}

Template literals are literals delimited with backtick (\`) characters, allowing for [multi-line strings](#multi-line_strings), for [string interpolation](#string-interpolation) with embedded expressions, and for special constructs called [tagged templates](#tagged-templates).

> Template literals are sometimes informally called _template strings_, but they aren't string literals and can't be used everywhere a string literal can be used. Also, a tagged template literal may not result in a string; it's up to the tag function what it creates (if anything).

Reviewer: ```suggestion
Template literals are sometimes informally called _template strings_, because one of their most common uses is to create strings by substituting placeholder variables and expressions and returning a string ([string interpolation](#string-interpolation)). 
Not however they ate are _not_ string literals, and can't be used everywhere a string literal can be used. In addition, a tagged template literal may not result in a string; it can be used with a custom [tag function](#tagged-templates) to create any type of object.
```

// not 20."
```

> That can make your code harder to read – especially when you have multiple expressions.

Reviewer: ```suggestion
That can be hard to read – especially when you have multiple expressions.
```
